src/admin.rs

```diff
@@ -439,3 +448,101 @@ where
 
     write_all_half(stream, res).await
 }
+
+/// Show currently connected clients
+async fn show_clients<T>(stream: &mut T) -> Result<(), Error>
+where
+    T: tokio::io::AsyncWrite + std::marker::Unpin,
+{
+    let columns = vec![
+        ("client_id", DataType::Text),
+        ("database", DataType::Text),
+        ("user", DataType::Text),
+        ("application_name", DataType::Text),
+        ("transaction_count", DataType::Numeric),
+        ("query_count", DataType::Numeric),
+        ("error_count", DataType::Numeric),
```

Comment:
Errors are pooler errors encountered by the client. In this PR I introduce two kinds
- Checkout Failure
- Server Connection Banned while client is talking to it